Tests/SwiftGRPCNIOTests/NIOServerTests.swift

```diff
@@ -300,3 +300,89 @@ extension NIOServerTests {
     waitForExpectations(timeout: defaultTimeout)
   }
 }
+
+// Only test Unary and ServerStreaming, as ClientStreaming is not
+// supported in HTTP1.
+extension NIOServerTests {
+  private func gRPCEncodedMessage(_ string: String) -> Data {
+    var data = string.data(using: .utf8)!
+    data.insert(contentsOf: [UInt8]([0x0a, UInt8(data.count)]), at: 0)
+    data.insert(contentsOf: [UInt8]([0x00, 0x00, 0x00, 0x00, UInt8(data.count)]), at: 0)
+    return data
+  }
+  
+  private func sendOverHTTP1(rpcMethod: String, message: String, handler: @escaping (Data?, Error?) -> Void) {
+    let serverURL = URL(string: "http://localhost:5050/echo.Echo/\(rpcMethod)")!
+    var request = URLRequest(url: serverURL)
+    request.httpMethod = "POST"
+    request.setValue("application/grpc-web-text", forHTTPHeaderField: "Content-Type")
```

Comment:
I assume `application/grpc-web-text` will be the more common use case than `application/grpc-web`?

Reply:
I think so? The Javascript library does `-text`. Still support for web could be improved/tested more. Want me to add a FIXME?

Reply:
Yes, please. Or just a `TODO`, whatever you think appropriate :-)

Reply:
Added